recipe/meta.yaml

```diff
@@ -20,33 +20,25 @@ source:
     - osx_configure.patch  # [osx]
 
 build:
-  number: 0
-  skip: True  # [win and py36]
-  features:
-    - vc9     # [win and py27]
-    - vc14    # [win and (py35 or py36)]
-  detect_binary_files_with_prefix: True
+  number: 1
 
 requirements:
   build:
-    - toolchain
-    - python         # [win]
-    - libtool        # [unix]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}  # [unix]
```

Comment:
FWIW `conda-build` doesn't respect this for some strange reason. ( https://github.com/conda/conda-build/issues/2958 )

Reply:
It's doing the right thing for these, though, right? One of the builds fails if the fortran compiler is selected on Windows because flang doesn't install with vc9.